### PR TITLE
Add @internal to what is not covered by our BC promise

### PR DESCRIPTION
**Pull request by leofeyer**
Implements #1031 

**TODO**

* [x] What about controllers? Do they need an `@internal` tag, too?
* [x] What about `Contao\CoreBundle\Repository\RememberMeRepository`?
* [x] What about `Contao\ManagerBundle\Api\*`?
* [x] What about `Contao\ManagerBundle\ContaoManager\ApiCommand\*`?

### CODE REVIEW

File: calendar-bundle/src/Picker/EventPickerProvider.php [hunk 1]
```diff
@@ -33,6 +33,9 @@ class EventPickerProvider extends AbstractInsertTagPickerProvider implements Dca
      */
     private $security;
 
+    /**
+     * @internal Do not inherit from this class; decorate the "contao.cache.clear_internal" service instead
```
Comment (Toflar) [inline calendar-bundle/src/Picker/EventPickerProvider.php]: I guess that's a mistake here?

**Review by Toflar — CHANGES_REQUESTED** [review]
LGTM, found one issue in the comments.
Regarding your questions:

* What about controllers? Do they need an @internal tag, too?

I don't see why they would need to be extended. So yes.

* What about Contao\CoreBundle\Repository\RememberMeRepository?

I don't see why repositories would need to be extended. So yes.

* What about Contao\ManagerBundle\Api\*?
* What about Contao\ManagerBundle\ContaoManager\ApiCommand\*?

Imho the whole manager bundle is internal. What do you need to use or extend there?

File: calendar-bundle/src/Picker/EventPickerProvider.php (hunk 1 above)
Comment (leofeyer) [reply calendar-bundle/src/Picker/EventPickerProvider.php]: Indeed. 🤦‍♂

Comment (leofeyer) [reply calendar-bundle/src/Picker/EventPickerProvider.php]: Fixed in 3c7da70f61f2474ea81b445cadb56cff27fe7383.

**Review by Toflar — APPROVED** [review]
LGTM. I'd like to have other approvals though.

**Comment by leofeyer** [comment]
It seems no-one else was interested in reviewing, therefore I have merged this one now. 🤷‍♂

**Comment by aschempp** [comment]
> It seems no-one else was interested in reviewing, therefore I have merged this one now. 🤷‍♂

That's not really true, looks like two days was not enough time 😉

**Pull request merged**